src/tictactoe/marker_validation.clj

```diff
@@ -0,0 +1,33 @@
+(ns tictactoe.marker-validation
```

Comment:
I feel like there are already too many namespaces for the amount of functionality you've implemented. There isn't really a 1-1 correspondence between namespaces and classes in Java and I feel like that's where you're headed.

Let's try this. Implement everything in a single namespace for now and when it becomes painful, or responsibilities start to become muddled we'll extract namespaces.